xcode/Subconscious/Shared/Components/Common/SubtextView.swift

```diff
@@ -10,11 +10,25 @@ import SwiftUI
 struct SubtextView: View {
     private static var renderer = SubtextAttributedStringRenderer()
     var subtext: Subtext
+    var transcludes: Dictionary<Slashlink, EntryStub>
+    var onViewTransclude: (MemoAddress) -> Void
 
     var body: some View {
         VStack(alignment: .leading) {
             ForEach(subtext.blocks, id: \.self) { block in
-                Text(Self.renderer.render(block.description))
+                if let link = block
+                        .slashlinks
+                        .get(0)?
+                        .toSlashlink(),
```

Comment:
This will currently _only_ render a transclude when a Slashlink appears on its own in a block.

Reply:
Should this cap out at a maximum? I _sometimes_ write blocks with upwards of 5 slashlinks in them and I can imagine 5 transcludes in a row feeling a little excessive.

Regardless, I'll make the change and then we can find out by playtesting.

Reply:
Yeah let's say max 3 for now.